.ci/Jenkinsfile

```diff
@@ -383,7 +383,9 @@ def generateFunctionalTestStep(Map args = [:]){
                       powershell label: "Check for running elasticsearch", script: "Test-NetConnection ${linuxIp} -Port 9200"
                       powershell label: "Check for running kibana", script: "Test-NetConnection ${linuxIp} -Port 5601"
                       powershell label: "Check for running fleet server", script: "Test-NetConnection ${linuxIp} -Port 8220"
-                      cmd script: "cd e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
+                      dir("${BASE_DIR}\\outputs") {
+                        cmd script: "cd ..\e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.format=\"junit:${BASE_DIR}/outputs/TEST-windows-${suite}.xml\" --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
```

Comment:
@v1v are we missing one slash at the beginning?

Reply:
yep :/ 

```suggestion
                        cmd script: "cd ..\\e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.format=\"junit:${BASE_DIR}/outputs/TEST-windows-${suite}.xml\" --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
```